pom.xml

```diff
@@ -51,19 +51,18 @@ The project was originally started in December 2002 by Arthur van Hoff at Strang
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
-		<jvm.version>1.8</jvm.version>
+		<maven.compiler.release>21</maven.compiler.release>
 	</properties>
-
 	<dependencies>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
-			<version>2.0.7</version>
+			<version>2.0.16</version>
 		</dependency>
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<version>4.13.1</version>
+			<version>4.13.2</version>
```

Comment:
Is this essential for this PR? Because it will introduce a merge conflict for the other PR https://github.com/jmdns/jmdns/pull/318. But ultimately the other PR makes this unnecessary.

Reply:
I can revert that its not necessary